client/asset/dcr/dcr.go

```diff
@@ -885,6 +939,39 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	return 1 + uint64(atomsPerKB)/1000, nil // dcrPerKB * 1e8 / 1e3
 }
 
+// externalFeeEstimator gets the fee rate from the external API
+func externalFeeEstimator(ctx context.Context, net dex.Network, nb uint64) (float64, error) {
+	var url string
+	if net == dex.Testnet {
+		url = testnetExternalApiUrl
+	} else {
+		url = externalApiUrl
```

Comment:
Note that for simnet it's gonna use mainnet's fee estimates.  This isn't bad TBH as it allows some more fee rate variability (maybe not with DCR, but in general it would).